packages/react-components/src/components/utils.ts

```diff
@@ -149,6 +151,25 @@ export const messageBarType = (errorType: ErrorType): MessageBarType => {
   }
 };
 
+/* @conditional-compile-remove(pinned-participants) */
```

Comment:
no need to remove this. it's being exported

Reply:
done